mira/metamodel/templates.py

```diff
@@ -112,14 +112,17 @@ class Config:
             SympyExprStr: lambda e: sympy.parse_expr(e)
         }
 
-    def with_context(self, do_rename=False, **context) -> "Concept":
+    def with_context(self, do_rename=False, name_map=None, **context) -> "Concept":
         """Return this concept with extra context.
 
         Parameters
         ----------
         do_rename :
             If true, will modify the name of the node based on the context
             introduced
+        name_map :
+            A mapping of context values to names. Useful if the context values
+            are e.g. curies.
```

Comment:
add in the docs what happens if you don't pass this vs what happens if you do

Reply:
Resolved in https://github.com/gyorilab/mira/pull/252/commits/fc93fcf176b6c14205c0b2a096733847d094c17f